### PR TITLE
Created new soccer page for e-commerce app

### PR DESCRIPTION
**Pull request by YevheniiaTykhenko**

### CODE REVIEW

File: applications/e-commerce/soccer.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                        <li>Soccer balls</li>
+                        <li>Ball accessories</li>
+                        <li>Soccer nets</li>
+                    </ul>    
+                </li>
+                <li>Soccer shoes</li>
+                <ul>
+                    <li>Adults</li>
+                    <li>Kids</li>
+                </ul>
```
Comment (tupes) [inline applications/e-commerce/soccer.html]: You should have the closing `</li>` after the `<ul>`, like you did with "Soccer balls and nets"

File: applications/e-commerce/soccer.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                <li>Soccer shoes</li>
+                <ul>
+                    <li>Adults</li>
+                    <li>Kids</li>
+                </ul>    
+                <li>Soccer clothing and equipment</li>
+                <ul>
+                    <li>Adults</li>
+                    <li>Kids</li>
+                </ul>
```
Comment (tupes) [inline applications/e-commerce/soccer.html]: Same as above.

File: applications/e-commerce/soccer.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en"> 
+    <head>
+        <meta charset="utf-8"> <title>Soccer</title>     
+    </head>
```
Comment (tupes) [inline applications/e-commerce/soccer.html]: Minor detail, but you should fix your formatting here 😄

**Review by tupes — CHANGES_REQUESTED** [review]
Awesome job!

If you make these changes, I'll definitely approve!

**Comment by YevheniiaTykhenko** [comment]
I made a changes in my file. Hope it will be ok.

**Review by tupes — APPROVED** [review]
Great!

**Pull request merged**